source

```diff
@@ -53283,7 +53309,9 @@ You cannot submit this form when the field is incorrect.</samp></pre>
   <hr>
 
   <p>The <dfn attribute for="HTMLInputElement"><code data-x="dom-input-list">list</code></dfn> IDL
-  attribute must reflect the <code data-x="attr-input-list">list</code> content attribute.</p>
+  attribute must return the current <span data-x="concept-input-list">suggestions source
+  element</span>, if any, <span data-x="dom-retarget">retargeted</span> against <span>this</span>,
+  or null otherwise.</p>
```

Comment:
I like the idea to use reflection, but I felt weird about it for two reasons:
1) _suggestions source element_ is not strictly the same as _list-associated element_, since it depends on the type of the list-associated element;
2) reflecting `list` to an Element means that the explicitly set list-element gets defined; no real impact in practice since it can never be set, but it felt weird.

For (1), it would be kind of nice if we had a generic mechanism to say "the attr-associated element should be type X, but the reflected element may not be", but I'm too tired to figure that out today!